java/src/jmri/NamedBean.java

```diff
@@ -76,18 +81,18 @@
      * @param listenerRef - A textual reference for the listener, that can be
      *                    presented to the user when a delete is called
      */
-    public void addPropertyChangeListener(java.beans.PropertyChangeListener l, String name, String listenerRef);
+    public void addPropertyChangeListener(@Nonnull java.beans.PropertyChangeListener l, String name, String listenerRef);
```

Comment:
Thanks for catching that.

The other call, addPropertyChangeListener(java.beans.PropertyChangeListener l, String beanRef, String listenerRef), works in AbstractNamedBean using the listener as a key.  So if want those two calls to be consistent, that will have to be rethought.